models/jules/inst/template.job

```diff
@@ -20,7 +20,7 @@ if [ ! -e "@OUTDIR@/*.nc" ]; then
   # check the status
   if [ $STATUS -ne 0 ]; then
     echo -e "ERROR IN MODEL RUN\nLogfile is located at '@OUTDIR@/logfile.txt'" >&3
-    exit $STATUS
+#    exit $STATUS
```

Comment:
This is, unfortunately, a hack. There's something wrong with how we're configuring dates in JULES that's causing the model to try and read meteorology for 1 year beyond the end_date. It can't find it so it throws an error, but it does this at the end of the run. If I let job.sh exit, then the model output (which is complete) never gets converted from model-specific format into PEcAn output standard. I've tried a bunch of things to try and solve the bug, and talked to Tris about it, but no luck so far -- still on my to-do list but I needed to get the runs done.  Once I sort that out I'll revert to throwing the error correctly.